nix/overrides/bazel/default.nix

```diff
@@ -135,7 +139,10 @@ stdenv.mkDerivation rec {
   sourceRoot = ".";
 
   patches = [
-    ./python-stub-path-fix.patch
+    # On Darwin, the last argument to gcc is coming up as an empty string. i.e: ''
+    # This is breaking the build of any C target. This patch removes the last
+    # argument if it's found to be an empty string.
+    ./trim-last-argument-to-gcc-if-empty.patch
```

Comment:
Do you know what changed to make this patch necessary? Given that the patch is also in upstream nixpkgs, don’t bother investigating if you don’t know offhand. Just curious if you know what’s going on.

Reply:
An issue with `proto_library` targets on Darwin, see https://github.com/NixOS/nixpkgs/pull/63879. I don't know the details, though.